internal/services/iothub/iothub_resource.go

```diff
@@ -602,6 +602,12 @@ func resourceIotHub() *pluginsdk.Resource {
 				},
 			},
 
+			"disable_local_auth": {
```

Comment:
within the provider we're exposing this as `local_authentication_enabled` for consistency (since some API's specify enabled and others use disabled) - can we update this to match?